lib/printer.js

```diff
@@ -568,9 +568,11 @@ function genericPrintNoParens(path, options, print) {
 
         if (n.argument) {
             var argLines = path.call(print, "argument");
-            if (argLines.length > 1 &&
-                namedTypes.JSXElement &&
-                namedTypes.JSXElement.check(n.argument)) {
+            if (argLines.isFirstLineOnlyComment() ||
```

Comment:
Got it. I'm happy to keep iterating with you on `Lines#isFirstLineOnlyComment`, in that case!